src/applications/disability-benefits/all-claims/pages/alternateNames.js

```diff
@@ -9,7 +9,7 @@ export const uiSchema = {
     'ui:widget': 'yesNo'
   },
   alternateNames: {
-    'ui:description': 'What names did you serve under?',
+    'ui:description': 'What name did you serve under?',
```

Comment:
This should've been part of the 1.2 PR - this change reflects the latest copy. Just FYI.